monero-outreach-docs/en/storyboard/refuge_en.md

```diff
@@ -0,0 +1,95 @@
+# Monero: Refuge From Grievous Civil Forfeiture Laws (Part 1)
+*04/15/19*

```

Comment:
Add two trailing spaces at the end of the strings to create a newline, please.

Reply:
Done